easybuild/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023a.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'JAGS'
+version = '4.3.2'
+
+homepage = 'http://mcmc-jags.sourceforge.net/'
+description = """JAGS is Just Another Gibbs Sampler.  It is a program for analysis
+ of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  """
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = [
+    ('https://sourceforge.net/projects/mcmc-%(namelower)s/files/%(name)s/%(version_major)s.x/Source/', 'download'),
+]
+sources = [SOURCE_TAR_GZ]
+checksums = ['871f556af403a7c2ce6a0f02f15cf85a572763e093d26658ebac55c4ab472fc8']
+
+configopts = ' --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'libexec/%(namelower)s-terminal', 'lib/libjags.so'],
```

Comment:
@jkuhl-uni please revert the removal of `SHLIB_EXT` here - that's the approach we use, which copes with different library naming conventions.

Reply:
Sure, thanks for the hint. I gained this EC from --try-toolchain and didn't pay attention, sorry.